evm/gas-service.js

```diff
@@ -304,24 +314,42 @@ async function processCommand(config, chain, options) {
             }
 
             try {
-                const tx = await timeout(
-                    gasService.updateGasInfo(chainsToUpdate, gasInfoUpdates, gasOptions),
-                    chain.timeout || 60000,
-                    new Error(`Timeout updating gas info for ${chain.name}`),
-                );
-
-                printInfo('TX', tx.hash);
-
-                const receipt = await timeout(
-                    tx.wait(chain.confirmations),
-                    chain.timeout || 60000,
-                    new Error(`Timeout updating gas info for ${chain.name}`),
-                );
-
-                const eventEmitted = wasEventEmitted(receipt, gasService, 'GasInfoUpdated');
-
-                if (!eventEmitted) {
-                    printWarn('Event not emitted in receipt.');
+                if (options.useRelay) {
+                    const gasLimit = Number(await gasService.estimateGas.updateGasInfo(chainsToUpdate, gasInfoUpdates));
```

Comment:
does gasLimit need to be set? will the API auto-estimate if we don't set it? That would be ideal, otherwise use the chain's gasLimit from config or a high value like 5M as a default. Otherwise we're relying on the RPC in our configs which might not be robust for this

Reply:
It should autoestimate if unset, yeah